lib/storage/metadata/mongoclient/MongoClientInterface.js

```diff
@@ -393,7 +419,8 @@ class MongoClientInterface {
         const versionId = generateVersionId(this.replicationGroupId);
         // eslint-disable-next-line
         objVal.versionId = versionId;
-        const vObjName = formatVersionKey(objName, versionId);
+        const vObjName = formatVersionKey(objName, versionId, params.vFormat);
+        const mObjName = formatMasterKey(objName, params.vFormat);
```

Comment:
take the chance to use more descriptive name : the `m` or `v` prefix looks like a minor "type" indicator, when it actually is most important...

```suggestion
        const versionKey = formatVersionKey(objName, versionId, params.vFormat);
        const masterKey = formatMasterKey(objName, params.vFormat);
```

Reply:
additionally, maybe it is be better to tweak the code a bit more, to compute both the versionKey and masterKey in one call?

since most parameters of both functions are actually the same, and both calls are made almost every time...

Reply:
> additionally, maybe it is be better to tweak the code a bit more, to compute both the versionKey and masterKey in one call?

I'm not really a fan of doing that, i think it'll just over complicate things. IMO we should keep them separate just to keep things intuitive